mqtt-transport/src/main/java/com/esri/geoevent/transport/mqtt/MqttInboundTransport.java

```diff
@@ -92,84 +86,72 @@ public void start() throws RunningException
 
 	@Override
 	public void run()
-	{
-		this.receiveData();
-	}
-
-	private void receiveData()
 	{
 		try
 		{
-			applyProperties();
-			setRunningState(RunningState.STARTED);
-
-			String url = (ssl ? "ssl://" : "tcp://") + host + ":" + Integer.toString(port);
-			mqttClient = new MqttClient(url, MqttClient.generateClientId(), new MemoryPersistence());
+			mqtt.applyProperties(this);
+			mqttClient = mqtt.createMqttClient(new MqttCallback()
+			{
 
-			mqttClient.setCallback(new MqttCallback()
+				@Override
+				public void messageArrived(String topic, MqttMessage message) throws Exception
 				{
-
-					@Override
-					public void messageArrived(String topic, MqttMessage message) throws Exception
+					try
 					{
-						try
-						{
-							receive(message.getPayload());
-						}
-						catch (RuntimeException e)
+						if (log.isTraceEnabled())
 						{
-							e.printStackTrace();
+							log.trace("messageArrived: " + topic + "(" + message + ")");
 						}
-					}
-
-					@Override
-					public void deliveryComplete(IMqttDeliveryToken token)
+						receive(message.getPayload());
+					} catch (RuntimeException e)
 					{
-						// not used
+						e.printStackTrace();
 					}
+				}
 
-					@Override
-					public void connectionLost(Throwable cause)
-					{
-						log.error("CONNECTION_LOST", cause.getLocalizedMessage());
-					}
-				});
+				@Override
+				public void deliveryComplete(IMqttDeliveryToken token)
+				{
+					// not used
+				}
 
-			MqttConnectOptions options = new MqttConnectOptions();
+				@Override
+				public void connectionLost(Throwable cause)
+				{
+					log.error("CONNECTION_LOST", cause.getLocalizedMessage());
+				}
+			});
 
-			// Connect with username and password if both are available.
-			if (username != null && password != null && !username.isEmpty() && password.length > 0)
-			{
-				options.setUserName(username);
-				options.setPassword(password);
-			}
+			mqttClient.subscribe(mqtt.getTopic(), mqtt.getQos());
 
-			if (ssl)
+			if (log.isTraceEnabled())
 			{
-				// Support TLS only (1.0-1.2) as even SSL 3.0 has well known exploits
-				java.util.Properties sslProperties = new java.util.Properties();
-				sslProperties.setProperty("com.ibm.ssl.protocol", "TLS");
-				options.setSSLProperties(sslProperties);
+				log.trace("Started MQTT Inbound Transport");
 			}
+			setRunningState(RunningState.STARTED);
 
-			options.setCleanSession(true);
-			mqttClient.connect(options);
-			mqttClient.subscribe(topic, qos);
-
-		}
-		catch (Throwable ex)
+		} catch (Throwable ex)
```

Comment:
formatting - the `catch` statement needs to start a new line:
```suggestion
		} catch (Exception e)
```
```suggestion
		} catch (Exception e)
```
````
try
{
  ...
}
  catch (...)
{
  ...
}
```